packages/data-point/lib/trace/trace-resolve.js

```diff
@@ -0,0 +1,72 @@
+const uniqueId = require('lodash/uniqueId')
+
+const NS_PER_SEC = 1e9
+
+/**
+ * @param {hrtime} time hrtime value
+ * @returns {Integer} nanosecond
+ */
+function hrtimeTotNanosec (time) {
+  return time[0] * NS_PER_SEC + time[1]
+}
```

Comment:
`lib/trace/trace-graph.js` also has functions like `nanoToMillisecond` and the `NS_PER_SEC` var. the helper code for manipulating hrtime should probably go in a single helper file instead of spreading it around.

Reply:
yup, I struggled with that one, but you are right

Reply:
btw @seoyoon94 had also mentioned this, so ... both of you thanks for pointing it out, sending an update soon